arcade/standard/bed_rush/map.xml

```diff
@@ -0,0 +1,587 @@
+    <team id="team-two" color="blue" max="40">Blue</team>
+</teams>
+<!-- observer spawn -->
+<spawns>
+    <default>
+        <regions>
+            <point yaw="90">20,34,0.5</point>
+        </regions>
+    </default>
+</spawns>
+<time result="score">15m</time>
+<world>
+    <timeset>18000</timeset>
+</world>
+<gamerules>
+    <doDaylightCycle>false</doDaylightCycle>
+</gamerules>
+<maxbuildheight>21</maxbuildheight>
+<!-- prefered gamemode is "bedrush" displayed as "Bed Rush" -->
+<gamemode>bridge</gamemode>
+<gamemode>bedwars</gamemode>
+<respawn auto="true" delay="0s"/>
+<constants>
+    <constant id="score-limit">5</constant>
+    <constant id="team-one-spawn">0.5,21,-38.5</constant>
+    <constant id="team-two-spawn">0.5,21,39.5</constant>
+</constants>
+<broadcasts>
+    <alert after="1s" filter="only-team-one">`7`lDestroy the Blue Team's Bed!</alert>
+    <alert after="1s" filter="only-team-two">`7`lDestroy the Red Team's Bed!</alert>
+    <alert after="10m">`7If neither team has won by then, the match will end in `c`l5 minutes`7!</alert>
+    <alert after="13m">`7If neither team has won by then, the match will end in `c`l2 minutes`7!</alert>
+    <alert after="14m">`7If neither team has won by then, the match will end in `c`l1 minute`7!</alert>
+    <alert after="14m30s">`7If neither team has won by then, the match will end in `c`l30 seconds`7!</alert>
+    <alert after="14m50s">`7If neither team has won by then, the match will end in `4`l10 seconds`7!</alert>
+</broadcasts>
+<spawns>
+    <spawn team="team-one" kit="spawn-kit" yaw="0" pitch="0" region="team-one-spawn-point"/>
+    <spawn team="team-two" kit="spawn-kit" yaw="180" pitch="0" region="team-two-spawn-point"/>
+</spawns>
+<kits>
+    <kit id="spawn-kit">
+        <clear/>
+        <item material="iron sword" enchantment="sharpness:1" unbreakable="true"/>
+        <item material="iron pickaxe" enchantment="efficiency:1" unbreakable="true"/>
+        <item material="iron axe" enchantment="efficiency:1" unbreakable="true"/>
+        <item material="golden apple" amount="8"/>
+        <item material="sandstone" damage="2" amount="64"/>
+        <item material="sandstone" damage="2" amount="64"/>
+        <item material="sandstone" damage="2" amount="64"/>
+        <item material="sandstone" damage="2" amount="64"/>
+        <item material="wood" amount="64"/>
+        <effect duration="6s" amplifier="255">resistance</effect>
+        <helmet material="leather helmet" enchantment="protection:2" team-color="true" unbreakable="true" prevent-sharing="true"/>
+        <chestplate material="leather chestplate" enchantment="protection:3" team-color="false" unbreakable="true" prevent-sharing="true"/>
+        <leggings material="leather leggings" enchantment="protection:2" team-color="true" unbreakable="true" prevent-sharing="true"/>
+        <boots material="leather boots" enchantment="protection:3;feather_falling:1" team-color="true" unbreakable="true" prevent-sharing="true"/>
```

Comment:
Did you meant `locked="true"` instead of `prevent-sharing="true"`? You would use prevent-sharing to stop people from putting their armors in a chest or elsewhere

Reply:
Sounds good, will update